asdf/_tests/_issues/test_1558.py

```diff
@@ -0,0 +1,34 @@
+import weakref
+
+import numpy as np
+import pytest
+
+import asdf
+
+
+def test_1558(tmp_path):
```

Comment:
Use a descriptive name for the test.

Reply:
This follows the convention set up in: https://github.com/asdf-format/asdf/pull/1537